org.gnome.seahorse.Application.json

```diff
@@ -0,0 +1,130 @@
+{
+    "app-id": "org.gnome.seahorse.Application",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "command": "seahorse",
+    "copy-icon": true,
```

Comment:
```suggestion
```
The icon name upstream seems to be set to the app-id, so no need to copy the icon here. It's only needed if you rename the icon and want to keep the icon with the original icon name in case it's used somewhere.

Reply:
TIL. Thanks!